src/js/global.js

```diff
@@ -77,6 +77,7 @@ ubuntu.globalNav = function() {
       let wrapper = createFromHTML(
         '<nav class="global-nav">' +
         '  <div class="global-nav__wrapper">' +
+        '    <h2 class="global-nav__title">Ubuntu websites</h2>' +
```

Comment:
This should be a ```<button>``` element with aria tags stating what it controls. Anything which is interactive but not a link is advised to be a button element.